pombola/core/static/js/functions.js

```diff
@@ -198,4 +198,71 @@ $(function () {
     e.preventDefault();
     location.href = "/search?q=" + escape($('#id_q,#loc').first().val());
   });
+
+  const showMessageToolsId = $("#show-message-tools-toggle")
+  const messagesContainer = $("#message-tips-container")
+
+  const pageResponsiveWidth = 991;
+  // hide element initially if page width is smaller than 991px
+  if (window.innerWidth < pageResponsiveWidth) {
+    messagesContainer.hide()
+  }
+
+  // check for window resize actions and toggle show message tools
+  window.addEventListener('resize', function(event){
+    if (window.innerWidth < pageResponsiveWidth) {
+      messagesContainer.hide()
```

Comment:
why are we using js for this and not css breakpoints?

Reply:
Good point, lets change this to CSS, better and more reliable across browsers than JS.